zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -133,7 +135,11 @@ public static boolean executePublish(Collection collection, CollectionReader col
                 if (imageFuture == null) {
                     throw new Exception("image future unexpectedly null on completion of publishing");
                 }
-                success = success && imageFuture.get();
+                ImageServicePublishingResult result = imageFuture.get();
+
+                if (result.getUnpublishedImages() != null && result.getUnpublishedImages().size() > 0) {
```

Comment:
Sorry I missed this before - but we should check `result != null` too. Otherwise `result.getUnpublishedImages()` will throw a null pointer ex.